spec/ttml1.xml

```diff
@@ -8055,16 +8068,19 @@ vocabulary defined by this specification, given the following
 constraints:</p>
 <ulist>
 <item>
-<p>The implicit duration of an anonymous span is defined as follows: if
-the anonymous span's parent time container is a parallel time container, then
+<p>The implicit duration of an anonymous span, a <el>set</el> element, or a <el>span</el> 
+element whose children 
+are exclusively text nodes, is defined as follows: if
+the span's parent time container is a parallel time container, then
 the implicit duration is equivalent to the <code>indefinite</code>
-duration value as defined by <bibref ref="smil21"/>; if the anonymous span's
+duration value as defined by <bibref ref="smil21"/>; if the span's
 parent time container is a sequential time container, then the implicit duration
 is equivalent to zero.</p>
 </item>
 <item>
-<p>The implicit duration of a <el>body</el>, <el>div</el>, <el>p</el>,
-or <el>span</el> element is determined in accordance to (1) whether
+<p>The implicit duration of a <el>body</el> element, <el>div</el> element, <el>p</el> element,
+or <el>span</el> element (but not of a <el>span</el> element whose children 
```

Comment:
Rather than the parentheses it would be consistent with the previous bullet, and clearer, to say "or `span` element whose children are not exclusively text nodes".